Resources/Textures/Objects/Misc/Business_card.rsi/meta.json

```
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Created by Aserovich (Discord)",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+        {
+            "name": "Synicate_card"
```

```suggestion
            "name": "syndicate_card"
```
Same for the file name.
The folder should be renamed to business_card.rsi